pkg/cmd/marathon/app/app_version_list.go

```diff
@@ -25,12 +28,54 @@ func newCmdMarathonAppVersionList(ctx api.Context) *cobra.Command {
 	cmd := &cobra.Command{
 		Use:   "list",
 		Short: "List the version history of an application.",
+		Args:  cobra.ExactArgs(1),
+		PreRunE: func(cmd *cobra.Command, args []string) error {
+			if cmd.Flags().Changed("max-count") && maxCount <= 0 {
+				return fmt.Errorf("maximum count must be a positive number")
+			}
+			return nil
+		},
 		RunE: func(cmd *cobra.Command, args []string) error {
-			return python.InvokePythonCLI(ctx)
+			client, err := marathon.NewClient(ctx)
+			if err != nil {
+				return err
+			}
+
+			versions, err := marathonAppVersionList(*client, args[0], maxCount)
+			if err != nil {
+				return err
+			}
+
+			// versions is a slice of strings that has to be printed like
+			// [
+			//   "val1",
+			//   "val2"
+			// ]
+			// To do so, we use the json package to marshal the slice
+			// and then print it as a string.
+			jsonVersions, err := json.MarshalIndent(versions, "", "  ")
+			if err != nil {
+				return err
+			}
+			fmt.Println(string(jsonVersions))
```

Comment:
I understand we don't want to change the output now but to me it doesn't make any sense to just spit out JSON to the user instead of a nicely-formatted output. In other commands we have `--json` for accomplishing this.